spec/intl.html

```diff
@@ -396,8 +396,7 @@ <h1>DateTime Format Functions</h1>
     </emu-clause>
 
     <emu-clause id="sec-formatdatetimepattern" aoid="FormatDateTimePattern">
-      <h1>FormatDateTimePattern ( _dateTimeFormat_, <ins>_pattern_</ins>, _patternParts_, <del>_x_</del><ins>_epochNanoseconds_</ins>, _rangeFormatOptions_ )</h1>
-
+      <h1>FormatDateTimePattern ( _dateTimeFormat_, <ins>_pattern_</ins>, _patternParts_, <del>_x_</del><ins>_epochNanoseconds_</ins>, _rangeFormatOptions_ <ins>[ , _timeZone_ ]</ins> )</h1>
```

Comment:
We should describe this new parameter in the paragraph below. I'd also consider calling it "timeZoneOverride" to clarify that it's meant to override the DateTimeFormat's internal time zone if it's present, but on the other hand that's less concise.

Reply:
I’ve added it in description, but didn’t change the parameter name, for conciseness. Let me know if you’d like me to change that.

Reply:
This is fine, the parameter name can always be updated some other time.